golang/_redirects

```diff
@@ -5,75 +5,73 @@
-/eventing-ceph/* go-get=1 /golang/eventing-ceph.html 200
-/eventing-contrib/* go-get=1 /golang/eventing-contrib.html 200
-/eventing-couchdb/* go-get=1 /golang/eventing-couchdb.html 200
-/eventing-istio/* go-get=1 /golang/eventing-istio.html 200
-/eventing-github/* go-get=1 /golang/eventing-github.html 200
-/eventing-gitlab/* go-get=1 /golang/eventing-gitlab.html 200
-/eventing-kafka/* go-get=1 /golang/eventing-kafka.html 200
-/eventing-kafka-broker/* go-get=1 /golang/eventing-kafka-broker.html 200
-/eventing-kogito/* go-get=1 /golang/eventing-kogito.html 200
-/eventing-natss/* go-get=1 /golang/eventing-natss.html 200
-/eventing-prometheus/* go-get=1 /golang/eventing-prometheus.html 200
-/eventing-rabbitmq/* go-get=1 /golang/eventing-rabbitmq.html 200
-/eventing-redis/* go-get=1 /golang/eventing-redis.html 200
-/func/* go-get=1 /golang/func.html 200
-/func-tastic/* go-get=1 /golang/func-tastic.html 200
-/hack/* go-get=1 /golang/hack.html 200
-/homebrew-client/* go-get=1 /golang/homebrew-client.html 200
-/homebrew-kn-plugins/* go-get=1 /golang/homebrew-kn-plugins.html 200
-/infra/* go-get=1 /golang/infra.html 200
-/kn-plugin-admin/* go-get=1 /golang/kn-plugin-admin.html 200
-/kn-plugin-diag/* go-get=1 /golang/kn-plugin-diag.html 200
-/kn-plugin-event/* go-get=1 /golang/kn-plugin-event.html 200
-/kn-plugin-func/* go-get=1 /golang/kn-plugin-func.html 200
-/kn-plugin-migration/* go-get=1 /golang/kn-plugin-migration.html 200
-/kn-plugin-operator/* go-get=1 /golang/kn-plugin-operator.html 200
-/kn-plugin-quickstart/* go-get=1 /golang/kn-plugin-quickstart.html 200
-/kn-plugin-sample/* go-get=1 /golang/kn-plugin-sample.html 200
-/kn-plugin-service-log/* go-get=1 /golang/kn-plugin-service-log.html 200
-/kn-plugin-source-kafka/* go-get=1 /golang/kn-plugin-source-kafka.html 200
-/kn-plugin-source-kamelet/* go-get=1 /golang/kn-plugin-source-kamelet.html 200
-/kn-plugin-source-pkg/* go-get=1 /golang/kn-plugin-source-pkg.html 200
-/knobots/* go-get=1 /golang/knobots.html 200
-/kperf/* go-get=1 /golang/kperf.html 200
-/monitoring/* go-get=1 /golang/monitoring.html 200
-/net-certmanager/* go-get=1 /golang/net-certmanager.html 200
-/net-contour/* go-get=1 /golang/net-contour.html 200
-/net-gateway-api/* go-get=1 /golang/net-gateway-api.html 200
-/net-http01/* go-get=1 /golang/net-http01.html 200
-/net-istio/* go-get=1 /golang/net-istio.html 200
-/net-kourier/* go-get=1 /golang/net-kourier.html 200
-/networking/* go-get=1 /golang/networking.html 200
-/operator/* go-get=1 /golang/operator.html 200
-/pkg/* go-get=1 /golang/pkg.html 200
+/actions/* go-get=:go-get /golang/actions.html 200
+/async-component/* go-get=:go-get /golang/async-component.html 200
+/caching/* go-get=:go-get /golang/caching.html 200
+/client/* go-get=:go-get /golang/client.html 200
+/client-pkg/* go-get=:go-get /golang/client-pkg.html 200
+/community/* go-get=:go-get /golang/community.html 200
+/container-freezer/* go-get=:go-get /golang/container-freezer.html 200
```

Comment:
should we delete this one since we're archiving the repo whenever I remember to actually do it?

Reply:
When you archive the repo - we can regenerate this config that will remove it.